second-edition/src/ch07-01-mod-and-the-filesystem.md

```diff
@@ -373,13 +560,23 @@ note: ... or maybe `use` the module `server` instead of possibly redeclaring it
 <span class="caption">Listing 7-5: Error when trying to extract the `server`
 submodule into *src/server.rs*</span>
 
+<span class="caption">Listagem 7-5: Erro ao tentar extrair o submódulo `server`
+ em *src/server.rs* </span>
+
 The error says we `cannot declare a new module at this location` and is
 pointing to the `mod server;` line in *src/network.rs*. So *src/network.rs* is
 different than *src/lib.rs* somehow: keep reading to understand why.
 
+O erro diz que `cannot declare a new module at this location` (não podemos declarar um novo módulo neste local) e é
+apontando para a linha `mod server;` em *src/network.rs*. Então *src/network.rs* é
```

Comment:
Acho melhor a expressão em inglês dentro dos parênteses.

Sugestão: "O erro diz que não podemos declarar um novo módulo neste local (`cannot declare a new module at this location`) e **está** apontando para a linha `mod server;` em *src/network.rs*."